apps/passport-server/scripts/scratch.ts

```diff
@@ -224,6 +227,48 @@ yargs
       }
     }
   )
+  .command(
```

Comment:
do not under any circumstances connect your local zupass code to the staging or production databases.